src/settings.c

```diff
@@ -252,14 +255,12 @@ void load_settings(char *cmdline_config_path)
         {
                 char *c = option_get_string(
                         "global",
-                        "follow", "-follow", "",
+                        "follow", "-follow", NULL,
```

Comment:
Default should be `defaults.f_mode`.

Reply:
Well, this wasn't the case before either, but I fully agree with you. New commit fixes it.